tests/virt_autotest/parallel_guest_migration_source.pm

```diff
@@ -163,6 +163,10 @@ sub guest_migration_test {
 
     $self->set_test_run_progress;
     my @guest_migration_test = split(/,/, get_var('GUEST_MIGRATION_TEST', ''));
+
+    # Remove 'virsh_live_native_p2p_manual_postcopy' if present and record a soft failure
+    @guest_migration_test = $self->filter_migration_tests(@guest_migration_test);
```

Comment:
You skip this test on all versions of host system ?

Reply:
yes, that's the issue, I forgot it. thanks for your reminder. will improve it.